webapp/src/css/material.less

```diff
@@ -31,7 +31,7 @@
     div {
       flex-grow: 100;
     }
-    &.age {
+    &.contact\.age {
```

Comment:
I'm worried that this will vary depending on the form, for example if some field has a field called "first-childs-age" I suspect this won't work.

Can you verify where the class `contact.age` is coming from? Can you think of a way to standardise it so all ages have the same css class regardless of their path?